src/ring_theory/ideal/basic.lean

```diff
@@ -590,6 +590,9 @@ def lift (S : ideal α) (f : α →+* β) (H : ∀ (a : α), a ∈ S → f a = 0
 @[simp] lemma lift_mk (S : ideal α) (f : α →+* β) (H : ∀ (a : α), a ∈ S → f a = 0) :
   lift S f H (mk S a) = f a := rfl
 
+def lift_mk_of_le (S T : ideal α) (H : S ≤ T) : S.quotient →+* T.quotient :=
```

Comment:
This name describes the implementation not the behavior. Perhaps something like:
```suggestion
/-- Weaken the relation of a quotient.

This is the `ideal.quotient` version of `quot.factor` -/
def inclusion (S T : ideal α) (H : S ≤ T) : S.quotient →+* T.quotient :=
```
would be better.